DDFIngestTest.py

```diff
@@ -0,0 +1,66 @@
+import sys, requests, re, time
+
+def main():
+    if len(sys.argv) != 4:
+        print_usage_statement()
+        return
+
+    ingest_type = sys.argv[1]
+    file_path = sys.argv[2]
+    xml_type = sys.argv[3]
+    file = open(file_path, "r")
+    file_contents = file.read()
+
+    requests.packages.urllib3.disable_warnings()
+    headers = {
+        "Content-Type" : "application/xml",
+        "Authorization" : "Basic YWRtaW46YWRtaW4="
+    }
+
+    url = "https://localhost:8993/services/csw/"
+
+    if ingest_type == "csw":
+        file_contents = generate_csw_transaction_xml(file_contents, xml_type)
+    elif ingest_type == "rest":
+        url = "https://localhost:8993/services/catalog?transform=" + xml_type
```

Comment:
We can do this in the future if it's needed.  If the intent is to test the speed of the transformation, CSW should suffice